### PR TITLE
Adding aliases to State content type

### LINKED ISSUE #501: Update endpoint: states
Create application/json alias

### PR DESCRIPTION
**Pull request by RyanM-RMA**
Adding integration tests for aliased content types 
Adding base case tests to FormatsTest, so we can confirm that the non-aliased values work.
Fixes #501

### CODE REVIEW

**Review by MikeNeilson — DISMISSED** [review]

File: cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java
Comment (rma-rripken) [inline cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java:1]: Its not necessary to refactor the restassured tests this much.     
The restassured tests should read:   given...when...then...assertThat

Comment (RyanM-RMA) [reply cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java:1]: Too much DRY for unit tests?  Do you want me to adjust it here, or leave the refactoring out going forward?

Comment (rma-rripken) [reply cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java:1]: I see how its cleaner but sometimes more is more.  I don't want to have to think about the tests - just want to read them.    I can see how for your three test cases having the refactored method makes it obvious what is duplicated/changed between the tests.  But then I probably wouldn't want theh ETag and cache-control test to get refactored

Comment (rma-rripken) [reply cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java:1]: Fix it now while its fresh and easy

Comment (MikeNeilson) [reply cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java:1]: Was an interesting idea, but it's good Ryan R spoke up, one does need to be careful how far you take DRY within test code.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**